src/dmd/backend/mscoffobj.d

```diff
@@ -409,19 +414,21 @@ version (SCPP)
  * pseg/offset to start of seg.
  */
 
-private extern (D)
-int32_t *patchAddr(int seg, targ_size_t offset)
+@trusted
+int32_t *MsCoffObj_patchAddr(int seg, targ_size_t offset)
 {
     return cast(int32_t *)(fobjbuf.buf + ScnhdrTab[SegData[seg].SDshtidx].PointerToRawData + offset);
 }
 
-private extern (D)
-int32_t *patchAddr64(int seg, targ_size_t offset)
+@trusted
+int32_t *MsCoffObj_patchAddr64(int seg, targ_size_t offset)
```

Comment:
Why is `private`  removed here?

Reply:
Also, why is this also changing the name of the function. Someone once said that refactorings should not be mixed with additions :P .

Reply:
The name was changed so it did not collide with one in machobj.d